src/components/LetterLoad.vue

```diff
@@ -22,6 +22,13 @@
         <v-card-text>
           <span v-html="letterBody" />
         </v-card-text>
+
+        <v-textarea
+          filled
+          auto-grow
+          placeholder="Additional message (optional)"
+          v-on:input="persistCustomization"
```

Comment:
I think this will work, but it is more common to use v-model for handling input. Doing that should let you trigger persistCustomization without an event.

Reply:
Thanks, Glenn @DietBepis1 .. I appreciate. I've made the fixes